docs/pages/project/contributing/contributing-docs.md

```diff
@@ -147,6 +147,25 @@ Meshery documentation is made of these components:
 
   _Note: From the Makefile, this command is actually running `$ bundle exec jekyll serve --drafts --livereload`. There are two Jekyll configuration, `jekyll serve` for developing locally and `jekyll build` when you need to generate the site artifacts for production._
 
+#### Note
+While performing the above step, if you're facing errors with message like below...
+
+`Your ruby version is x.x.x but your Gemfile specified 2.7.x`
+
+This is because Jekyll always consider the exact version of Ruby unlike JS
+
+So, you need to follow either of the two steps to resolve this problem
```

Comment:
A colon would be a nice ending to the statement above

Reply:
Resolved...